docs/device-agent/running.md

```diff
@@ -63,8 +63,10 @@ Web UI Options
 
 Setup command
 
-  -o, --otc string   Setup device using a one time code
-  -u, --ff-url url   URL of FlowFuse. Required for setup
+  -o, --otc string         Setup device using a one time code
+  -s, --otc-dont-start     Do not start the agent after setup
+  -n, --otc-dont-import    Do not ask to import Node-RED flows during setup
```

Comment:
```suggestion
  -s, --otc-no-start     Do not start the agent after setup
  -n, --otc-no-import    Do not ask to import Node-RED flows during setup
```

I may be off, but I think the pattern of `no-XYZ` is more common for negative flags like this.

Reply:
Have no spotted the `--dont-start` flag is not new - just missing from the docs... but I think we're still in the window of opportunity to change it if we want.

Reply:
I agree with proposal & will make the changes to both (and alter it in FF PR too)